File: Modul 2/SOAL-2/PRAK202_2410817220015_Ashma_Salimah.py
nilai1 = float(input("Masukkan Nilai Pertama : "))
nilai2 = float(input("Masukkan Nilai Kedua : "))
hasil = nilai1 + nilai2
if nilai1.is_integer():
    print(f"Hasil dari penjumlahan nilai pertama \"{int(nilai1)}\" dan nilai kedua \"{nilai2:.1f}\" adalah \"{hasil:.2f}\"")
else : 
    print(f"Hasil dari penjumlahan nilai pertama \"{nilai1:.2f}\" dan nilai kedua \"{nilai2:.1f}\" adalah \"{hasil:.2f}\"")

File: Modul 2/SOAL-2/PRAK202_2410817220015_Ashma_Salimah.c
#include <stdio.h>
int main() {
    float nilai1, nilai2;
    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &nilai1);
    printf("Masukkan Nilai Kedua : ");
    scanf("%f", &nilai2);
    float hasil = nilai1 + nilai2;
    if(nilai1==(int)nilai1) {
        printf("Hasil dari penjumlahan nilai pertama \"%d\" dan nilai kedua \"%.1f\" adalah \"%.2f\"", (int)nilai1, nilai2, hasil); }
    else { printf("Hasil dari penjumlahan nilai pertama \"%.2f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"", nilai1, nilai2, hasil); }
    return 0;
}